frontend/next-config/redirects/part-collections.js

```diff
@@ -27,9 +27,10 @@ function getLegacyPartItemTypeRedirects() {
 }
 
 function getRegexForTag(oldTag) {
-   return `(${escapeStringForRegex(
-      encodeURIComponent(destylizeURIComponent(oldTag))
-   )}|${escapeStringForRegex(oldTag)})`;
+   const encodedTag = encodeURIComponent(destylizeURIComponent(oldTag));
+   return encodedTag === oldTag
+      ? escapeStringForRegex(oldTag)
+      : `(${escapeStringForRegex(encodedTag)}|${escapeStringForRegex(oldTag)})`;
```

Comment:
Wouldn't this regex cover both cases so the ternary operator would not be needed?
- `encodedTag` **equals** `oldTag`
- `encodedTag` **does not equal** `oldTag`

Reply:
Yes this is to reduce the regex where we can. While `(Hello|Hello)` would technically work, i think we should just reduce it to `Hello` where we can.